<launch>
    <arg name="robot_namespace" value="robot_1"/>
    <!--       -->
    
    <node pkg="fyp_api" ns="$(arg robot_namespace)" type="exploration_module" name="frontier_module" output="screen">
    	<rosparam command="load" file="$(find fyp_api)/config/robot_config.yaml" />
    </node>    
    
</launch>


